lib/diagnostic.rb

```diff
@@ -105,15 +123,25 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < Cat
+include Carnivorous
+def initialize (roar)
+  @roar = "ROAR!"
+end
+end
+
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+inheritance is used for "is-a " situations and mixins
```

Comment:
The primary advantage is that classes can included any number of modules, but can only inherit from a single class.